src/Core/Services/OpenAPI/OpenApiDocumentor.cs

```diff
@@ -61,7 +62,7 @@ public class OpenApiDocumentor : IOpenApiDocumentor
         /// </summary>
         /// <param name="sqlMetadataProvider">Provides database object metadata.</param>
         /// <param name="runtimeConfigProvider">Provides entity/REST path metadata.</param>
-        public OpenApiDocumentor(IMetadataProviderFactory metadataProviderFactory, RuntimeConfigProvider runtimeConfigProvider)
+        public OpenApiDocumentor(IMetadataProviderFactory metadataProviderFactory, RuntimeConfigProvider runtimeConfigProvider, HotReloadEventHandler<CustomEventArgs> handler)
```

Comment:
how is the handler used here?

Reply:
was leftover from splitting branches, should be fixed now